pom.xml

```diff
@@ -147,6 +147,11 @@
 					</exclusion>
 				</exclusions>
 			</dependency>
+			<dependency>
+				<groupId>org.apache.commons</groupId>
```

Comment:
Are you sure we need this dependency?

Reply:
To use this StringUtils.countMatches, I needed it 
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html

Reply:
Did you get any error when you did not have this?

Reply:
I did update the code